src/presence_analyzer/tests.py

```diff
@@ -53,6 +62,67 @@ def test_api_users(self):
         self.assertEqual(len(data), 2)
         self.assertDictEqual(data[0], {u'user_id': 10, u'name': u'User 10'})
 
+    def test_presence_weekday_wrong_id(self):
```

Comment:
It's not in test_data.csv, so the user doesn't exist. I guess I should make this more descriptive?